demo/src/app/components/+timepicker/demos/arrowkeys/arrowkeys.html

```diff
@@ -0,0 +1,6 @@
+<timepicker [(ngModel)]="myTime2" [arrowkeys]="false" [readonlyInput]="!isEnabled"></timepicker>
```

Comment:
readonly? initial idea was to disable enable the possibility to use arrow keys or mouse wheel. So it should look like this `[arrowkeys]="allowArrowKeys"`